src/core/vdom.ts

```diff
@@ -1236,6 +1236,8 @@ function wrapFunctionProperties(id: string, properties: any) {
 	return props;
 }
 
+type EventMapValue = { proxy: EventListener; callback: Function; options: any } | undefined;
```

Comment:
Could we type options? `passive: boolean` or similar?

Reply:
I wanted to actually use `EventListenerOptions` but I think the type didn't have passive. I'll type it though.